app/public/css/styles.css

```diff
@@ -21,6 +21,7 @@
   --color-pink: #e72076;
   --color-grey-dark: #20323f;
   --color-grey: #606a71;
+  --color-grey-medium: rgba(96,106,113, .6);
```

Comment:
Let's have this as `hex` like the rest of the colors, `#99A0A4 `.

Reply:
Fixed